docs/index.rst

```diff
@@ -335,6 +335,10 @@ You need to specify
 
     "Very Interesting Description"
 
+    cap-client metadata get basic_info.analysis_proponents.-2 -p bf6b8501822c4d2ba46028611354df7e
```

Comment:
i'm not sure if we want to have negative indexes here, as may be confusing, especially that we use json patch syntax on update, which understands '-', as the end of the list.. @ioannistsanaktsidis?

Reply:
@atrisovi  I agree with  @annatrz , this can be misleading for a user as  the '-' might not be so clear. We should think of another way or just give the possibility to get the full object and not the last part or 2nd before the last.

Reply:
ok cool thanks :)